prep-html-syntax/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title> HTML Syntax </title>
+    <body>
+      <h1 style="color:Orange;"> My favorite anime character is Zenitsu! </h1>
+      <img src="https://wallpaper.dog/large/20504184.jpg" alt="Zenitsu One Trick Pony" style="width:900px;height:600px">
+      <p style="color:Blue;"> He is extremely overpowered and has no clue because he sleeps through his major fights </p>
+    </body>
```

Comment:
Hi Ignacio, be sure to make the `<head></head>` and `<body></body>` elements siblings instead of nested.

```html
<!DOCTYPE html>
<html>
  <head>
  </head>
  <body>
  </body>
</html>
```